### ISSUE TITLE
QOL improvement: Lets get rid of cursed weapons

### ISSUE THREAD

**Issue by otebski** [torilmud/issues#277]
I think it's time to get rid of cursed weapons. 
Fumble, which was only meaningful application of cursed weapons, has been gone for quite a bit.

The mechanic is not  fun and does not add much to game-play.

(at the same time please have failed disarms put weapons into inventory. I know sinking khanjaris are hilarious but we got rid of all other equipment sinks)

**Comment by CountVooku** [comment]
I curse my spellbooks so I can't put them in a container with 'put all XXXX'....

**Comment by Ruagh** [comment]
Its simple. Failed disarm putting item into inventory instead of the ground will easily solve the issue. And I do agree that it is annoying today.

**Comment by shevarash** [comment]
I tend to agree, but would like to look into it more before committing to a direction. Leaving this open for now.
